report/src/sections/04-theory/06-termination.tex

```diff
@@ -0,0 +1,20 @@
+\subsection{Termination}\label{subsec:termination}
+
+In this subsection we will argument for our analysis terminating.
+
+Our analysis takes base in the semantics for instructions $\abssem{I}$.
+Hence, to prove that our analysis terminates, we would need to show that $\abssem{I}$ always reaches a fixed point.
```

Comment:
This is almost correct... our goal of the analysis is to find some fixed point of a function that is given by the constraint described in abstract interpretation in preliminaries, so we need to show that this function has a fixed point. 

This function however is monotone if the abstract semantics are monotone.